docs/az-aro-python-development.md

```diff
@@ -60,6 +60,15 @@ Tests can be run as follows:
 azdev test aro (--live) (--lf) (--verbose) (--debug)
 ```
 
+> An issue was discovered on macOS when running tests due to additional security to restrict multithreading in macOS High Sierra and later versions of macOS. \
+> If getting the following error:
+```
++[__NSCFConstantString initialize] may have been in progress in another thread when fork() was called.
+```
+>Add this variable to your env or add it to your profile to make it permanent in `~/.bash_profile` or `~/.zshrc`:\
+`export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES`\
+
+
```

Comment:
```suggestion
> An issue was discovered on macOS when running tests due to additional security to restrict multithreading in macOS High Sierra and later versions of macOS.
> If getting the following error:
> ```
> +[__NSCFConstantString initialize] may have been in progress in another thread when fork() was called.
> ```
> Add this variable to your env or add it to your profile to make it permanent in `~/.bash_profile` or `~/.zshrc`:
`export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES`


```

Reply:
Should be better now. Check it out @m1kola